corosync_metrics_test.go

```diff
@@ -30,30 +30,27 @@ func TestQuoromMetricParsing(t *testing.T) {
 		Nodeid      Votes Name
 	1084780051          1 dma-dog-hana01 (local)
 	1084780052          1 dma-dog-hana02
-	dma-dog-hana01:~ # 
 	`
-	getQuoromClusterInfo()
 	voteQuorumInfo, quorate, _ := parseQuoromStatus([]byte(quoromStatus))
 
-	if voteQuorumInfo["expectedVotes"] != 232 {
+	if voteQuorumInfo["expected_votes"] != 232 {
```

Comment:
here Dunno if `foo_bar` is good in golang hashes sinnce the var naming convention is `fooBar`

Reply:
same as https://github.com/ClusterLabs/ha_cluster_exporter/pull/66#discussion_r336948397

Reply:
yeo